cmd/lakefs/cmd/users.go

```diff
@@ -0,0 +1,101 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/cmd/lakefs/application"
+	"github.com/treeverse/lakefs/pkg/auth"
+	"github.com/treeverse/lakefs/pkg/auth/model"
+	"github.com/treeverse/lakefs/pkg/config"
+	"github.com/treeverse/lakefs/pkg/logging"
+	"github.com/treeverse/lakefs/pkg/stats"
+	"github.com/treeverse/lakefs/pkg/version"
+)
+
+type User struct {
+	userName        string
+	accessKeyID     string
+	secretAccessKey string
+}
+
+var errInvalidUserPropertyCommandLineArg = errors.New("invalid user property command line arg")
+
+type UserCreator = func(context.Context, *auth.DBAuthService, *auth.DBMetadataManager, *User) (*model.Credential, error)
+
+func getUserPropertyFromCmdArgs(cmd *cobra.Command, propertyName string) (*string, error) {
+	propertyValue, err := cmd.Flags().GetString(propertyName)
+	if err != nil {
+		return nil, fmt.Errorf("%w %s: %s", errInvalidUserPropertyCommandLineArg, propertyName, err)
```

Comment:
Golint would cause a failure "do not trigger dynamic error", I had to specifically to add a new error. Let me try to commit your suggestion and see what happens, maybe my code was slightly different